src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -357,15 +363,22 @@ getAccount cAddr = do
     balance <- getBalance =<< decodeCAddressOrFail cAddr
     return $ CAccount cAddr balance
 
+getWalletAccAddrsOrThrow :: WalletWebMode ssc m => CAddress -> m [CAddress]
+getWalletAccAddrsOrThrow wCAddr =
+    getWalletAccounts wCAddr >>= maybe noWallet return
```

Comment:
You can use [`whenNothingM`](https://github.com/serokell/universum/blob/f2ccf8afd862e37ccd204c0ef9efde48a05c2d29/src/Monad/Maybe.hs#L39) here